### PR TITLE
Replace some wrong character

### PR DESCRIPTION
**Pull request by Oliverlin67**
Replace a part of simple Chinese character to traditional Chinese and remove duplicate content

### CODE REVIEW

File: locales/zh_TW/zh_TW.json [hunk 1]
```diff
@@ -2,58 +2,57 @@
...
+    "American Samoa": "美屬薩摩亞",
+    "An error occured while uploading the file.": "上傳文件時發生錯誤。",
+    "An unexpected error occurred and we have notified our support team. Please try again later.": "發生意外錯誤，我們已通知我們的支持團隊。 請稍後再試.",
+    "Andorra": "安道爾",
     "Angola": "安哥拉",
     "Anguilla": "安圭拉",
-    "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "自加载此页面以来，另一个用户已更新此资源。 请刷新页面，然后重试。",
-    "Antarctica": "南极洲",
-    "Antigua And Barbuda": "安提瓜和巴布达",
-    "Antigua and Barbuda": "Antigua and Barbuda",
```
Comment (caouecs) [inline locales/zh_TW/zh_TW.json:46]: This tag is needed

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you but you have deleted a tag

File: locales/zh_TW/zh_TW.json (hunk 1 above)
Comment (Oliverlin67) [reply locales/zh_TW/zh_TW.json:46]: added

**Review by overtrue — APPROVED** [review]

**Comment by caouecs** [comment]
thank you

**Pull request merged**